af/fastpath/fastpath/core.py

```diff
@@ -742,15 +744,16 @@ def score_measurement_whatsapp(msm):
         scores["accuracy"] = 0.0
         return scores
 
-    if not tk.get("requests", []):
+    requests = tk.get("requests", ()) or ()
+    if not requests:
```

Comment:
It makes sense to consider it zero accuracy in the case of whatsapp, but not in the case of web_connectivity.

👍